shimmingtoolbox/cli/mask.py

```diff
@@ -169,9 +169,11 @@ def sphere(fname_input, fname_output, radius, center, verbose):
                                         "value is equal or less than this threshold. (default: 30)")
 @click.option('-v', '--verbose', type=click.Choice(['info', 'debug']), default='info', help="Be more verbose")
 def threshold(fname_input, output, thr, verbose):
-
+    
     # Set all loggers
     set_all_loggers(verbose)
+    # test
+    logger.info("this works third time")
```

Comment:
```suggestion
```

Reply:
Thanks! addresed this in [d8d4cd6](https://github.com/shimming-toolbox/shimming-toolbox/commit/d8d4cd68f1bacd99fccb915d8bb47d7ea3b53621)